src/pybind/mgr/dashboard/frontend/src/app/shared/components/language-selector/language-selector.component.spec.ts

```diff
@@ -26,14 +30,44 @@ describe('LanguageSelectorComponent', () => {
 
   it('should read current language', () => {
     expect(component.selectedLanguage).toBe('en-US');
+    expect(listLocales()).toEqual([]);
+  });
+
+  const expectLanguageChange = (lang) => {
+    component.changeLanguage(lang);
+    expect(component.selectedLanguage).toBe(lang);
+    expect(listLocales().includes(lang.slice(0, 2))).toBe(true);
+  };
+
+  it('should change to cs', () => {

```

Comment:
These tests test the same thing in multiple `it()` functions. I'd like to suggest one instead.